requirements.txt

```diff
@@ -1,9 +1,10 @@
 python-fedora>=0.3.7
```

Comment:
Now that the dependencies are in `pyproject.toml`, you can remove this file.

Reply:
done!